cpp/src/pca/pca_mg.cu

```diff
@@ -200,11 +204,11 @@ void fit_impl(raft::handle_t& handle,
     Stats::opg::mean_add(input_data, input_desc, mu_data, comm, streams, n_streams);
   }
 
-  for (int i = 0; i < n_streams; i++) {
+  for (std::size_t i = 0; i < n_streams; i++) {
```

Comment:
same here wrt 32-bit. Just a nitpick on this one because it shouldn't affect perf, but it does help readability.